sigh

```diff
@@ -0,0 +1,2 @@
+#!/bin/sh
+exec $(dirname $0)/tools/sigh "$@"
```

Comment:
Is this just for convenience? some concern that if we add this it'll confuse folks since we also have docs saying to call it via ./tools/sigh, we could update those docs. Do we really need the convenience, folks can add 'alias', etc in their shells instead, open to thought.

Reply:
Fair. I just found it convenient, it shouldn't really be part of this PR.